tests/integration/test_sharepoint.py

```diff
@@ -29,7 +29,13 @@ def FILE_NAME(sharepoint):
     os.remove(path)
 
 
-def test_credentials():
+def test_credentials_not_found():
```

Comment:
Please add docstring with raises section

Reply:
We are checking here if the exception is raised and not raising a new exception.